en/docs/administer/key-managers/configure-forgerock-connector.md

```diff
@@ -0,0 +1,288 @@
+# Configure Forgerock as a Key Manager
+
+WSO2 API Manager has the capability to support multiple Key Managers at the same time. So with the use of connectors, it is capable of supporting any authorization server as a Key Manager.
+Additionally it is prepacked with an inbuilt resident Key Manager.
+
+WSO2 API Manager can connect to Forgerock out-of-the-box using the [WSO2 API-M Forgerock Connector](https://github.com/wso2-extensions/apim-km-forgerock).
+
+Follow the instructions given below to configure Forgerock as a third-party Key Manager:
+
+### Configure Forgerock Server to work with APIM
+We assume that you already have a Forgerock installation and follow the steps below to configure the Fogerock Authorization server to work with WSO2 API Manager.
+
+#### Connfigure Oauth2 Provider
+1. Go to the URL where the Forgerock Authorization server is deployed and <br> navigate to 
+   Realms -> Top Level Realm. Top level realm which exists by default is the root realm and the examples and configurations in this guide are based on that.
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-realms.png)
+
+2. Navigate to Root Realm -> Dashboard -> Configure Oauth provider -> Configure Ouath 2.0 and create an oauth provider by filling up the values.
```

Comment:
As we are giving the user an example in this page, always make sure to give the sample values that they should add in text form that getting the user to look at the image and type out the values.